### PR TITLE
Add markdown tag

### PR DESCRIPTION
**Pull request by l-vincent-l**
Partial solution for #22

### CODE REVIEW

File: client/javascripts/app.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import riot from 'riot'
 import './components/**/*.tag'
+import './tags/*.tag'
```
Comment (maukoquiroga) [inline client/javascripts/app.js]: Why don't you just use /components/

Comment (l-vincent-l) [reply client/javascripts/app.js]: I don't know it's not really a component to me, but I can do that

File: client/javascripts/components/markdown/markdown.tag [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import showdown from 'showdown'
```
Comment (maukoquiroga) [inline client/javascripts/components/markdown/markdown.tag]: Does it work with just `import { Converter } from 'showdown' ?

File: client/javascripts/components/markdown/markdown.tag [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <div></div>
+
+    <script type="es6">
+        this.set = () => {
+            this.root.childNodes[0].innerHTML = convert(this.opts.content)
+        }
+        this.on('update', this.set)
+        this.on('mount', this.set)
+
+        var convert = (markdown) => {
```
Comment (maukoquiroga) [inline client/javascripts/components/markdown/markdown.tag]: ```javascript```
this.converter = new showdown.Converter()

this.convert = () => {
    this.root.childNodes[0].innerHTML = this.converter.makeHtml(this.opts.content)
}

this.on('update', this.convert)
this.on('mount', this.convert)
```

**Comment by maukoquiroga** [comment]
@l-vincent-l What do you think of:

```javascript
import showdown from 'showdown'

<markdown>
    <script type="es6">
        this.converter = new showdown.Converter()
        this.element   = document.createElement('div')
        this.element.setAttribute('class', 'markdown')
        this.root.appendChild(this.element)

        this.markdownify = (markdown) => {
            return this.converter.makeHtml(markdown)
        }

        this.parse = () => {
            this.element.innerHTML = this.markdownify(this.opts.content)
        }

        this.on('mount', this.parse)
        this.on('update', this.parse)
    </script>
</markdown>
```

**Comment by maukoquiroga** [comment]
> Use more **explicite** riot syntax

Should be **explicit**, could you rebase?

**Comment by l-vincent-l** [comment]
Nop, It doesn't worth the cost of rebasing https://github.com/etalab/transport-site/pull/52 and possibly add errors.

**Comment by maukoquiroga** [comment]
Rebase and GTM

**Pull request merged**